torch/_torch_docs.py

```diff
@@ -4759,7 +4761,9 @@ def merge_dicts(*dicts):
 
 Keyword arguments:
     {out}
-    {dtype}
+    dtype (torch.dtype, optional) : the desired data type of returned tensor. Default: if None,
+    and start and end are real values, uses a global default (see torch.get_default_tensor_type())
+    else sets it to the complex default type.
```

Comment:
For clarity, I think we could re-write "the complex default type" as "the complex type corresponding to the global default type".